docs/fundamentals/code-analysis/quality-rules/ca1068.md

```diff
@@ -61,70 +61,21 @@ public void LongRunningOperation(string usefulParameter, CancellationToken token
 
 If the method is an externally visible public API that is already part of a shipped library, then it is safe to suppress a warning from this rule to avoid a breaking change for the library consumers.
 
-## Configurability
+## Configure code to analyze
 
-This rule has the following configurable options that can be configured for just this rule, for all rules, or for all rules in this category (Design). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
-
-### Include or exclude selected APIs
+You can configure this option for just this rule, for all rules, or for all rules in this category (Design). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
 
 Use the following options to configure which parts of your codebase to run this rule on.
 
-- [Include specific API surfaces](#include-specific-api-surfaces)
+- [Select API to analyze](#select-api-to-analyze)
```

Comment:
I actually like the original heading (Include specific API surfaces) better. What is the reason to change it?